src/librustc_typeck/error_codes.rs

```diff
@@ -1588,6 +1588,38 @@ fn bar(foo: Foo) -> u32 {
 ```
 "##,
 
+E0183: r##"
+Fn* traits (`unboxed closures`) are experimental. 
+See [tracking issue #29625][iss29625] for the status of the feature.
+
+Functions must have exactly one (non self) argument, a tuple representing
```

Comment:
I'd put "self" between backquotes.

Reply:
Forgot to do this one with the last commit...